jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -81,6 +86,8 @@
   private final SkaffoldParameters skaffold;
   private final Property<Boolean> allowInsecureRegistries;
   private final Property<String> containerizingMode;
+  private final ListProperty<ExtensionParameters> pluginExtensions;
```

Comment:
Do we still need this, or are we fine with just `extensionParametersSpec`?

Reply:
This is passed to `extensionParametersSpec`, which in turns adds elements to this list.

BTW, I noticed the example in the class comment is out of date. Will update it.